src/NzbDrone.Core/Notifications/CustomScript/CustomScriptSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+using NzbDrone.Core.Validation.Paths;
+
+namespace NzbDrone.Core.Notifications.CustomScript
+{
+    public class CustomScriptSettingsValidator : AbstractValidator<CustomScriptSettings>
+    {
+        public CustomScriptSettingsValidator()
+        {
+            RuleFor(c => c.Path).IsValidPath();
+        }
+    }
+
+    public class CustomScriptSettings : IProviderConfig
+    {
+        private static readonly CustomScriptSettingsValidator Validator = new CustomScriptSettingsValidator();
+
+        [FieldDefinition(0, Label = "Path", Type = FieldType.Path)]
+        public String Path { get; set; }
+
+        [FieldDefinition(0, Label = "Arguments", HelpText = "Arguments to pass to the script")]
+        public String Arguments { get; set; }
```

Comment:
Ppl will share scripts, args allow the user to configure it without having to edit it.